README.md

```diff
@@ -1,3 +1,11 @@
 # Angular2-Meteor application
 
 Building Angular2 UI for a Meteor app : recruitment website.
+
+#Steps to run the app
+
+1. Clone this repo via HTTP or SSH into your machine
+2. cd into `hiring-meteor`
+3. run `meteor npm install`
```

Comment:
We are now using meteor 1.1 so the `meteor npm install` part is now available. That will be a future development.